projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3362,9 +3361,6 @@ public function admin_init() {
 		}
 
 		// Add custom column in wp-admin/users.php to show whether user is linked.
```

Comment:
This line should probably also be removed? Or is there something else that's going to replace these removed hooks?

Reply:
Yes sir. I moved this logic into sso.php since it doesn't make sense when SSO is off.